.vscode/settings.json

```diff
@@ -1,7 +0,0 @@
-{
-	"editor.insertSpaces": false,
-	"editor.tabSize": 4,
-	"search.exclude": {
-		"waterbox/**": true
-	}
-}

```

Comment:
Is this necessary for using VS Code? It shouldn't be included in this PR either way.

Reply:
Yeah, these sneaked into the repo without me noticing. Removed.

Reply:
No, those files are already checked-in, and you've deleted them. If you restore them, does VS Code stop working correctly?

Reply:
Those were added in https://github.com/TASEmulators/BizHawk/commit/0b14755ac303533819138a8b7f4aa8f02c780f0c.  This PR shouldn't be modifying them; if we do want to change what we provide for developers and their IDEs, use a separate PR for that.

Reply:
Agh I thought it was me who had added them. I never intended to change/add/remove these files. I'll see to put them back to their original state

Reply:
fixed